core/server/web/api/v0.1/routes.js

```diff
@@ -11,8 +11,7 @@ const cors = require('../../shared/middlewares/api/cors');
 const brute = require('../../shared/middlewares/brute');
 
 // Handling uploads & imports
-const tmpdir = require('os').tmpdir;
-const upload = require('multer')({dest: tmpdir()});
+const upload = require('../../shared/middlewares/upload');
```

Comment:
Had to enable this middleware for v0.1 because themes controller is still shared as well as importer service